ze/tracer_ze_helpers.include.c

```diff
@@ -887,6 +959,11 @@ static void thapi_sampling_energy() {
       readEnergy(i, j, &ts_us, &energy_uj);
       do_tracepoint(lttng_ust_ze_sampling, gpu_energy, (ze_device_handle_t)_sampling_hDevices[i], j, (uint64_t)energy_uj, ts_us);
     }
+    readPerformance(i, computeE, copyE);
+    for (uint32_t k=0; k<_sampling_subDeviceCount; k++){
+      do_tracepoint(lttng_ust_ze_sampling, computeEngine , (ze_device_handle_t)_sampling_hDevices[i], k, computeE[k].computeActive, computeE[k].timestamp);
+      do_tracepoint(lttng_ust_ze_sampling, copyEngine, (ze_device_handle_t)_sampling_hDevices[i], k, copyE[k].copyActive, copyE[k].timestamp);
+    }
```

Comment:
Use `tracepoint` here for now. After,  guard data fetching and tracepoint emission behind tracepoint_enabled and do_tracepoint.